github/messages_test.go

```diff
@@ -359,6 +359,13 @@ func TestParseWebHook(t *testing.T) {
 			payload:     &RepositoryDispatchEvent{},
 			messageType: "repository_dispatch",
 		},
+		{
+			payload:     &WorkflowDispatchEvent{},
+			messageType: "workflow_dispatch",
+		}, {
```

Comment:
Was this file run through `gofmt` ?

I'm really surprised it didn't catch this.

Could you please change this to make it more consistent with the rest of the repo?

```
},
{
```

Reply:
it was, but now it should be fine.